sys/proto.c

```diff
@@ -0,0 +1,158 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * sys/proto.c
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Ziemowit Leszczynski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#include <netdb.h>
+#include <string.h>
+#include <stdlib.h>
+
+
+#define PROTOCOLS_NUM 55
```

Comment:
It's better and safer to get this number by using `sizeof(protocols) / sizeof(*protocols)`